lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -128,15 +128,62 @@ def _create_video_with_subs(self):
         )
 
 
-class TestEmptyCourseVideoSummaryList(MobileAPITestCase):
+class TestNonStandardCourseStructure(MobileAPITestCase):
     """
     Tests /api/mobile/v0.5/video_outlines/courses/{course_id} with no course set
     """
     REVERSE_INFO = {'name': 'video-summary-list', 'params': ['course_id']}
 
-    def test_chapter_is_none(self):
+    def _verify_paths(self, course_outline, path_list):
         """
-        Tests when there is no chapter under course, and video under course
+        Takes a path_list and compares it against the course_outline
+
+        Attributes:
+            path_list (list): A list of the expected strings
+            course_outline (list): A list of dictionaries that includes a 'path'
+                and 'named_path' field which we will be comparing path_list to
+        """
+        path = course_outline[0]['path']
+        self.assertEqual(len(path), len(path_list))
+        for i in range(0, len(path_list)):
+            self.assertEqual(path_list[i], path[i]['name'])
+        #named_path will be deprecated eventually
+        named_path = course_outline[0]['named_path']
+        self.assertEqual(len(named_path), len(path_list))
+        for i in range(0, len(path_list)):
+            self.assertEqual(path_list[i], named_path[i])
```

Comment:
Nit: performance doesn't matter in this code, but consider including L154 within the top `for` loop so you have `N` loops instead of `2N` loops.